certifikit/src/commonMain/kotlin/app/cash/certifikit/platform.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package app.cash.certifikit
+
+import okio.ByteString
+import okio.IOException
+
+internal expect fun ByteString.toBitList(unusedBitsCount: Int): List<Int>
+
+expect class ProtocolException(message: String? = null) : IOException
+
+expect class BigInteger
+
+internal expect fun BigInteger.toByteArray(): ByteArray
+
+internal expect fun BigInteger.testBit(bit: Int): Boolean
+
+internal expect fun BigInteger.bitLength(): Int
+
+internal expect fun BigInteger.toHexString(): String
+
+internal expect fun ByteArray.toBigInteger(): BigInteger
+
+internal expect fun String.parseUtcTime(): Long
+
+internal expect fun Long.formatUtcTime(): String
+
+internal expect fun Long.formatGeneralizedTime(): String
+
+internal expect fun String.parseGeneralizedTime(): Long
+
+// TODO: the modality of these 3 is worryingly platform specific
+
+expect abstract class X509Certificate
```

Comment:
I don’t think we want a class with this name in the ultimate result. Maybe just `Certificate` ? (Are there other kinds?)

Reply:
We use Certificate in certifikit.  So maybe PlatformCertificate?

Reply:
Maybe we don't offer an expect class for all platforms, and instead offer platform-specific converters? `Certificate.toX509()` on the JVM, and similar on other platforms?